### ISSUE TITLE
Examples from README.md are broken with v1.1.2

### ISSUE THREAD

**Issue by adioanca** [exocet-engineering/hfx-bus#5]
### With v1.1.1, the example works. Only v1.1.2 has problems.

1. start consumer
2. start committer => nothing happens.
3. stop both
4. start consumer => lots of exceptions:

```
Bus S4KaCspAW1-3f-KU started!
{ event: 'healthz:ping:pending',
  asyncListener: [AsyncFunction],
  error:
   { Error: Undefined payload key "ping"
       at Object.message.load (/home/adi/leaflet/cc-live-map/node_modules/hfxbus/lib/hfx-bus.js:318:12)
       at process._tickCallback (internal/process/next_tick.js:68:7) data: { key: 'ping', message: [Object] } },
  args:
   [ { messageId: '_kpGUaDD0oiG_OKk',
       trackerId: 'yVBM5NZLjPPb27ZQ',
       streamId: '1557251758314-0',
       replyTo: [Array],
       groupName: 'healthz',
       streamName: 'ping',
       isAcknowledged: false,
       isResolved: null,
       _release: [Function: _release],
       ack: [AsyncFunction: ack],
       load: [AsyncFunction],
       save: [AsyncFunction],
       drop: [AsyncFunction],
       purge: [AsyncFunction],
       resolve: [AsyncFunction],
       reject: [AsyncFunction],
       ping: null } ] }
{ event: 'healthz:ping:pending',
  asyncListener: [AsyncFunction],
  error:
   { Error: Undefined payload key "ping"
       at Object.message.load (/home/adi/leaflet/cc-live-map/node_modules/hfxbus/lib/hfx-bus.js:318:12)
       at process._tickCallback (internal/process/next_tick.js:68:7) data: { key: 'ping', message: [Object] } },
  args:
   [ { messageId: '8AQzDFL3atkk4R7U',
       trackerId: 'ZuGDFH7V3RxN_vcb',
       streamId: '1557251763305-0',
       replyTo: [Array],
       groupName: 'healthz',
       streamName: 'ping',
       isAcknowledged: false,
       isResolved: null,
       _release: [Function: _release],
       ack: [AsyncFunction: ack],
       load: [AsyncFunction],
       save: [AsyncFunction],
       drop: [AsyncFunction],
       purge: [AsyncFunction],
       resolve: [AsyncFunction],
       reject: [AsyncFunction],
       ping: null } ] }
[...]
```

**Please complete the following information:**
 - OS and Version: Ubuntu 16.04
 - NodeJS Version: 10.15.3
 - Redis Version: 5.0.4
 - Redis Driver and Version: ioredis 4.9.3

**Comment by vflopes** [comment]
Thanks for reporting this bug, I'll try to reproduce the bug to see what is happening!

**Comment by vflopes** [comment]
Fixed at **v1.2.0** :smiley: 

Can you test again to see if the problem persists?
